test/tests/transfers/general_field/nearest_node/regular/main.i

```diff
@@ -84,8 +84,6 @@
     from_multi_app = sub
     source_variable = to_main
     variable = from_sub
-    # Some equidistant nodes
-    search_value_conflicts = false
```

Comment:
With equidistant nodes, what does it choose? Is it desirable?

Reply:
I m re-enabling the default here. 

With equidistant nodes it is undetermined, it chooses the lowest PID one by default